app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<h2>Welome! please Log in if you have an account already else Sign up ?</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name),class:'login-form') do |f| %>
+ <div class="form-group">
+  <div class="field-input">
+    <%= f.email_field :email, autofocus: true, placeholder:'Username', autocomplete: "email" %>
```

Comment:
- [ ] The placeholder here should be `email`, not `Username` for a better user experience